### DISCUSSION TITLE
Include organizations calls into openapi file

### DISCUSSION THREAD

**Discussion by Florzl** [keycloak/keycloak#33680] [Ideas]
Hello Keycloak Team,

today I looked at the new v26 api spec and found no calls related to organizations, which came by default with the latest update. I would like to get an organization or invite users to an organization via the api. Currently there are the exsisting calls in Keycloak (Chrome Dev Tools) but there is no documentation on this....

Could this be added in the future?
Thanks a lot!

**Comment by JonasDev17** [comment]
https://github.com/keycloak/keycloak/pull/34215
